app/Models/Entry.php

```diff
@@ -391,18 +390,9 @@ public static function getContentByParsing($url, $path, $attributes = array()) {
 		}
 
 		if ($html) {
+			require_once(LIB_PATH . '/lib_phpQuery.php');
 			$doc = phpQuery::newDocument($html);
 			$content = $doc->find($path);
-
-			foreach (pq('img[data-src]') as $img) {
```

Comment:
This was a workaround for one specific fix, which does not exist anymore https://github.com/FreshRSS/FreshRSS/pull/888